gradio/blocks.py

```diff
@@ -2434,6 +2436,7 @@ def reverse(text):
             print(
                 f"Monitoring URL: {self.local_url}monitoring/{self.app.analytics_key}"
             )
+        self.enable_monitoring = enable_monitoring in [True, None]
```

Comment:
I think we can just have this parameter be a bool and default to `True`? Might be clearer than `bool | None` - the only difference with having default `None` is that the link won't get printed to the console automatically but I don't think it's a bad thing if it does.

Reply:
See comment above